### ISSUE TITLE
Unable to reproduce results

### ISSUE THREAD

**Issue by supersubscript** [Shujun-He/RNAdegformer#4]
Hi, thanks for providing this model. I'm struggling with replicating the results using the files provided in the repository. Some of the paths do not seem to exist (e.g. the pseudo-labels), and the final model weights used in the paper don't seem to be available. Would you be able to upload these and add a short tutorial on how to reproduce everything?

**Comment by supersubscript** [comment]
Yeah, can't get the pseudolabelling to work. From where do the long/short preds/stds come from? The format doesn't match the output of `predict.py`, so I'm lost. @Shujun-He are you able to help?

**Comment by f-krause** [comment]
Hi @supersubscript! Did you come up with a solution in the end? I am trying to replicate the work as well right now. Would appreciate any insights, if I come up with a solution in the meantime I will share it.

**Comment by supersubscript** [comment]
Hi, @f-krause! I had to spend quite a bit of time refactoring the code-base to get something running, but I couldn't replicate the stated performance in the end. The authors have published some new work - you may be interested in looking at https://github.com/Shujun-He/RibonanzaNet

**Comment by f-krause** [comment]
Thanks a lot for the hint @supersubscript! I will check it out. Would you mind sharing the (probably predict.py) file you refactored to obtain the pseudo labels?

**Comment by supersubscript** [comment]
Err, I had a look, but I don't think I have that code around still. I did a pretty massive rehaul of the code though, so not just `predict.py`. The pseudolabelling part in particular is not well documented, so in some instances I had to make assumptions.

Sorry I can't help more than this. I think looking at the ribonanzanet code may set you off in the right direction though.

Best of luck!

**Comment by f-krause** [comment]
No worries, your input is very much appreciated! Thanks! :) 
I now constructed some way to create the pseudo labels but also making several assumptions. If results get close to the paper, I might create a PR here to provide a possible solution to this issue.

**Comment by supersubscript** [comment]
Sounds great. If it's of any help I think my reactivity R2 for the main dataset was around 0.59, but I didn't spend much time trying to push it, and I can't quite remember whether that was with CV or not

**Comment by f-krause** [comment]
I created a PR with my work on replicating the results in the paper. By making my own assumptions on how to generate the pseudo labels, I could obtain very similar results to He et al.

Feel free to check out my PR #5.

**Comment by h-ahl** [comment]
Thanks for this, @f-krause - great work! :)
